addons/website/static/src/components/dialog/page_properties.js

```diff
@@ -194,8 +195,9 @@ export class PagePropertiesDialog extends FormViewDialog {
         });
     }
 
-    deletePage() {
+    async deletePage() {
         const pageIds = [this.resId];
+        const fields = await this.orm.read("website.page", pageIds, ["is_new_page_template"]);
```

Comment:
I looked around and didn't find another way to do it so let's keep it like that :wink: 

Maybe just change the name to something like `newPageTemplateFields` so we know what these fields really are :slightly_smiling_face: